webpack.common.js

```diff
@@ -1,4 +1,7 @@
+const path = require('path');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const HtmlWebpackTemplate = require('html-webpack-template');
```

Comment:
I added 'html-webpack-plugin' and 'html-webpack-template' here so it does not have to be added to both `webpack.dev.js` or `webpack.prod.js` . Let me know if I should not have done this.

Reply:
This makes sense, `webpack.common.js` then gets merged into `dev` and `prod`.